outputs.tf

```diff
@@ -6,6 +6,14 @@ output "app_name" {
   value = var.app_name
 }
 
+output "humio_protocol" {
+  value = var.humio_protocol
+}
+
+output "humio_host" {
+  value = var.humio_host
+}
```

Comment:
(nit) do you need these outputs? I mean it's just passing back the input variables?

Reply:
I was following the AWS Terraform module pattern where all inputs are exposed as outputs. I believe this allows developers to create dependencies between resources so that certain resources are built before others.